### PR TITLE
feat(query): added "CloudTrail Log Files S3 Bucket Logging Disabled" for Terraform

### PR DESCRIPTION
**Pull request by rafaela-soares**
**Proposed Changes**
- Added "CloudTrail Log Files S3 Bucket Logging Disabled" for Terraform (missing severity and severity)

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by rogeriopeixotocx** [comment]
This is pending severity and category that will be added soon

File: assets/queries/terraform/aws/cloudtrail_log_files_s3_bucket_with_logging_disabled/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "ee9e50e8-b2ed-4176-ad42-8fc0cf7593f4",
+  "queryName": "CloudTrail Log Files S3 Bucket with Logging Disabled",
+  "severity": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/cloudtrail_log_files_s3_bucket_with_logging_disabled/metadata.json:4]:
```suggestion
  "severity": "HIGH",
```

File: assets/queries/terraform/aws/cloudtrail_log_files_s3_bucket_with_logging_disabled/metadata.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "category": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/cloudtrail_log_files_s3_bucket_with_logging_disabled/metadata.json:5]:
```suggestion
  "category": "Observability",
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/aws/cloudtrail_log_files_s3_bucket_with_logging_disabled/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "queryName": "CloudTrail Log Files S3 Bucket with Logging Disabled",
+    "severity": "",
```
Comment (rogeriopeixotocx) [inline assets/queries/terraform/aws/cloudtrail_log_files_s3_bucket_with_logging_disabled/test/positive_expected_result.json:4]:
```suggestion
    "severity": "HIGH",
```

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]

**Pull request merged**